README.md

```diff
@@ -52,6 +52,15 @@ var httpResponse = oauthClient.authorizationCodeGrant(URI.create("https://login.
 
 If your authorization server doesn't allow wildcards, you can also configure a fixed path (and even port) via e.g. `setRedirectPath("/callback")` and `setRedirectPorts(8080)` before calling `authorize(...)`.
 
+### Client Credentials Grant
```

Comment:
Should there be a note in this chapter or in the code snippet that a user can optionally set scopes or should the use have a look into the method signature or method documentation? I can live with either

Reply:
I can certainly add this, same applies for Authorization Code Grant

Reply:
added in 8028a7f